src/styles/components/price.scss

```diff
@@ -0,0 +1,24 @@
+@import "./utilits/mixins.scss";
+
+.price {
+  &__text {
+    color: $secondary-color;
+  }
+
+  &__number {
+    margin-bottom: 16px;
+    font-size: 16px;
+    @include text-bold;
+  }
+}
+
+.reviews {
```

Comment:
The `reviews` does not belong to the `price` block, so you should not keep them in one file.